client.py

```diff
@@ -159,21 +144,18 @@ def filter_git_commit_data(repository_name, repository_team, repository_version)
     number = 0
     repository_path = repository_team + repository_name
     print("Working on repo: {}, Latest release: {}".format(repository_name, repository_version))
-    repo_dict.update({number: {"lastChecked": str(datetime.datetime.utcnow()),
+    repo_dict.update({number: {"lastChecked": str(datetime.utcnow()),
                                "last_two_Release": repository_version}})
 
     try:
-        #x = type(repository_version['LatestRelease']['Date'])
-        x = datetime.datetime.strptime(repository_version['LatestRelease']['Date'], '%a, %d %b %Y %H:%M:%S GMT')
-        #y = repository_version['PreviousRelease']['Date']
-        #h = datetime.strptime(x, '%Y-%m-%d %H:%M:%S')
-        #c = datetime.strptime(y, '%Y-%m-%d %H:%M:%S')
-        y = datetime.datetime.strptime(repository_version['PreviousRelease']['Date'], '%a, %d %b %Y %H:%M:%S GMT')
-        z = x - y
+        latestRelease = datetime.strptime(repository_version['LatestRelease']['Date'], '%a, %d %b %Y %H:%M:%S GMT')
+        previousRelease = datetime.strptime(repository_version['PreviousRelease']['Date'], '%a, %d %b %Y %H:%M:%S GMT')
+        timeDifference = latestRelease - previousRelease
+        lastCheck = datetime.now() - timeDifference
```

Comment:
We want to get the changes since the previous version until the latest version.
What this code do is to get the commits since (datetime.now - difference between last two versions)
e.g for release services: latest version - previous version = ~ 15 days
so it will print the commits since (datetime.now - 15 days) including the latest commits which are not released